azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/setup/components/SetupControlBarView.kt

```diff
@@ -196,7 +199,7 @@ internal class SetupControlBarView : LinearLayout {
     }
 
     private fun openAudioDeviceList() {
-        viewModel.displayAudioDeviceSelectionMenu()
+        openAudioDeviceSelectionMenuCallback()
```

Comment:
Same thing, can we remove `openAudioDeviceList()` and  have the OnClickListener directly call `openAudioDeviceSelectionMenuCallback()`